### DISCUSSION TITLE
Rotated and overlapped images to text in epub

### DISCUSSION THREAD

**Discussion by graizzaro** [sumatrapdfreader/sumatrapdf#3189] [Q&A]
In this epub https://github.com/sumatrapdfreader/sumatrapdf/files/10298387/0.Model.making.for.architects.2013.zip (I deleted some images to reduce file dimensions), some images are 180° rotated and overlapped with text 

See this SumatraPdf captured image of the problem
![SumatraPdf](https://user-images.githubusercontent.com/73635134/209432436-293b14bf-b8f3-4047-8578-5d71ec1ae138.jpg)

Calibre don't show it
![Calibre](https://user-images.githubusercontent.com/73635134/209432439-21a39b20-5575-4559-9930-6914cd33e831.jpg)

I suspect that depends from css configuration ( probably width-50 and similar commands )

Can you help me to solve this problem?

**Comment by GitHubRulesOK** [comment]
Seems like its a buggy rendering by the MuPDF engine, nothing odd about css as seen by other apps (looks fine otherwise)
![image](https://user-images.githubusercontent.com/12754913/209436008-8c3f182b-b9c3-4809-a7cc-75d1425966ad.png)
rendering in MS Edge etc
![image](https://user-images.githubusercontent.com/12754913/209436151-d022e188-a2c4-434b-a94b-a5a264d7d702.png)

**Comment by kjk** [comment]
I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3260 to track this
